Source/Rows/SliderRow.swift

```diff
@@ -116,6 +117,7 @@ open class SliderCell: Cell<Float>, CellType {
     }
 
     func valueChanged() {
+        guard !row.isDisabled else { return }
```

Comment:
Yep, thats right. If the slider is not enabled, then this method will never be called.